/gazebo_ros_control:
    pid_gains:
    #   Pid values removed for more stable movement
        Joint1: {p: 200.0, i: 40.0, d: 1000.0, antiwindup: true}
        Joint2: {p: 100.0, i: 25.0, d: 100.0, antiwindup: true}
        Joint3: {p: 200.0, i: 40.0, d: 105.0, antiwindup: true}
        Joint4: {p: 60.0, i: 5.0, d: 10.0, i_clamp_max: 10.0}
        Joint5: {p: 100.0, i: 20.0, d: 13.0, antiwindup: true}
        Gripper: {p: 50.0, i: 12.5, d: 15.0, antiwindup: true}
        # the following gains are used by the gazebo_mimic_joint plugin
        right_finger_joint: {p: 100.0, i: 0.01, d: 10.0, i_clamp: 0.2, antiwindup: true, publish_state: true}
        left_finger_joint: {p: 100.0, i: 0.01, d: 10.0, i_clamp: 0.2, antiwindup: true, publish_state: true}

lynxmotion_al5d:
    # Publish all joint states -----------------------------------
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50  

    # Controller for all joints
    joints_positions:
        type: position_controllers/JointGroupPositionController
        joints:
            - Joint1
            - Joint2
            - Joint3
            - Joint4
            - Joint5
            - Gripper
